core/imageroot/etc/systemd/system/send-backup.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=Store the encrypted cluster backup remotely
 ConditionPathExists=/var/lib/nethserver/cluster/state/backup/passphrase
+After=apply-updates.service
```

Comment:
Why this should be needed? I'd do not bind together the actions, updated info will be send the day after.

Reply:
This should ensure that the service does not during updates to avoid conflicts.